app/styles/_core.less

```diff
@@ -115,6 +115,7 @@
 .breadcrumb {
   font-size: @font-size-base - 1;
   padding-bottom: 0;
+  margin-bottom: 0;
```

Comment:
the bottom margin on breadcrumbs didn't actually have an effect with the h1 as the next element, and looks fine when the alerts show up